ruby/test/Constants_Test.rb

```diff
@@ -0,0 +1,30 @@
+########################################################################################################################
+#  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+#  See also https://openstudio.net/license
+########################################################################################################################
+
+require 'minitest/autorun'
+
+class ConstantsTest < Minitest::Test
+
+  # Note: putting all of these tests under the same one because of the way we
+  # run the tests with the CLI (we run a CLI instance for each individual test,
+  # and it takes almost 20 seconds for each which is ridiculous)
+  def test_ruby_constants
```

Comment:
new test. Grouping everything under a single minitest because it is WWAAAAAY to slow otherwise with the CLI.